tests/functional/d/distutil_regression_import_error.py

```diff
@@ -0,0 +1,4 @@
+# pylint: disable=missing-docstring
```

Comment:
We might as well add a module docstring + link to some of the relevant issue 😄 

Saves us the trouble of looking them up again when we need to revisit

Reply:
I agree. These once could be useful
* https://github.com/PyCQA/pylint/issues/73
* https://github.com/PyCQA/pylint/issues/2955
* https://github.com/PyCQA/astroid/pull/1321

Reply:
```suggestion
"""
Regression test to check that distutils can be imported
See https://github.com/PyCQA/pylint/issues/73

See also:
https://github.com/PyCQA/pylint/issues/2955
https://github.com/PyCQA/astroid/pull/1321
"""
```